packages/techdocs-common/src/stages/publish/local.ts

```diff
@@ -84,6 +84,7 @@ export class LocalPublish implements PublisherBase {
           .catch(reason => {
             reject(reason);
           });
+        fs.remove(directory);
```

Comment:
This is indeed useful. We don't really need the generated directory once it has been published to a more non-temporary location.